mail_tracking/models/ir_mail_server.py

```diff
@@ -12,6 +13,8 @@ class IrMailServer(models.Model):
     def _tracking_headers_add(self, tracking_email_id, headers):
         """Allow other addons to add its own tracking SMTP headers"""
         headers = headers or {}
+        headers['X-Odoo-Database'] = getattr(
+            threading.currentThread(), 'dbname', None),
```

Comment:
Don't you have any other place where getting the db name, like `self.env.cr.database.name` or similar?